src/Orchard.Web/Modules/Orchard.Blogs/Views/Content-BlogPost.SummaryAdmin.cshtml

```diff
@@ -25,17 +28,17 @@
             @T(" | ")
 
             if (contentItem.HasDraft()) {
-                @Html.Link(T("Publish Draft").Text, Url.Action("Publish", "Admin", new { area = "Contents", id = contentItem.Id, returnUrl = Request.ToUrlString() }), new { itemprop = "UnsafeUrl" })
+                @Html.Link(T("Publish Draft").Text, Url.BlogPostPublish(post), new { itemprop = "UnsafeUrl" })
```

Comment:
@sebastienros for one, the helper already exists in `Orchard.Blogs.Extensions.UrlHelperExtensions`, and I couldn't see it being used anywhere else, so... why not? :smile:  

More importantly though, Publish in BlogPostAdmin controller checks the BlogPostPublish permission, while Publish in the standard Contents AdminController checks the regular PublishContent permission. So for normalizing on the Blog specific permissions, it's important.